salt/cli/batch.py

```diff
@@ -26,6 +26,59 @@
 log = logging.getLogger(__name__)
 
 
+def _batch_get_opts(
+        tgt,
+        fun,
+        parent_opts,
+        arg=(),
+        tgt_type='glob',
+        ret='',
+        kwarg=None,
+        batch='10%',
```

Comment:
I would have exepected both the `tgt_type` and `batch` to be required args instead of kwargs with defaults. Is there a reason they're specified here in the interface such as they are?

Reply:
@dincamihai Did you see the feedback here?

Reply:
@cachedout @rallytime I took those params defaults as they were here defined here: https://github.com/saltstack/salt/blob/d4b28f5aa4f928a1c95e4989aadb09400818e90a/salt/client/__init__.py#L497-L506

Reply:
@cachedout I can make them required args and have the defaults comming from cmd_batch, that's not a problem

Reply:
@cachedout done https://github.com/saltstack/salt/pull/49818/commits/9a5513ae00133cbf843deb95b23c755a30f50ed8